shared/js/browser/common.js

```diff
@@ -6,16 +6,30 @@
  *   featureSettings?: import('../../../schema/__generated__/schema.types').RemoteFeatureSettings | undefined
  * }} BackgroundTabData
  */
+
 /**
  * @module common
  */
+
+/**
+ * Returns content height of nav view based on topmost subview.
+ * If the topmost subview contains a data-max-view-height attribute, it caps the height at that value.
+ *
+ * @returns {number|null|undefined}
+ */
 export const getContentHeight = () => {
-    const $openSubviewV2 = window.document.querySelector(
-        '#popup-container.sliding-subview-v2--root [data-current]:last-of-type > *:first-child'
+    const $openSubviewV2 = /** @type HTMLElement */ (
```

Comment:
This is my solution for a requirement to cap the height of the desktop popover/modals on a specific view. The view sets data-max-view-height and the mutationObserver returns whichever height is smaller.